docs/sources/setup-grafana/configure-security/configure-scim-provisioning/manage-users-teams/_index.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Learn how to implement SCIM provisioning in Grafana for automated user and team synchronization. SCIM integrates with identity providers like Okta and Azure AD to streamline user management, automate team provisioning, and replace Team Sync.
+keywords:
+  - grafana
+  - scim
+  - provisioning
+  - user-management
+  - team-management
+labels:
+  products:
+    - cloud
+    - enterprise
+menuTitle: Manage users and teams with SCIM
+title: Manage users and teams with SCIM
+weight: 310
+---
+
+# Manage users and teams with SCIM
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{< /admonition >}}
+
+SCIM streamlines identity management in Grafana by automating user lifecycle and team membership operations. This guide explains how SCIM works with existing Grafana setups, handles user provisioning, and manages team synchronization.
+
+With SCIM, you can:
+
+- **Automate user lifecycle** from creation to deactivation
+- **Manage existing users** by linking them with identity provider identities
+- **Synchronize team memberships** based on identity provider group assignments
+- **Maintain security** through automated deprovisioning
+- **Replace Team Sync** with more robust SCIM group synchronization
+
+## User provisioning with SCIM
+
+SCIM provisioning works in conjunction with existing user management methods in Grafana. While SCIM automates user provisioning from the identity provider, users can still be created through SAML just-in-time provisioning when they log in, manually through the Grafana UI, or via automation tools like Terraform and the Grafana API. For the most consistent user management experience, we recommend centralizing user provisioning through SCIM.
+
+For detailed configuration steps specific to the identity provider, see:
+
+- [Configure SCIM with Azure AD](../configure-scim-azure/)
+- [Configure SCIM with Okta](../configure-scim-okta/)
+
+### How SCIM identifies users
+
+SCIM uses two identifiers to establish and maintain user identity between the identity provider and Grafana:
+
+1. Email address:
+
+   - Used for initial user matching
+   - Can change over time
+   - Included in SAML NameID
+
+2. Identity provider user ID:
+   - Unique, permanent identifier from the identity provider
+   - Stored as `externalId` in SCIM
+   - Must match between SAML assertions and SCIM
+   - Prevents security issues from email changes
+
+### Existing Grafana users
+
+{{< admonition type="note" >}}
+Existing users must be assigned to the Grafana app in the identity provider to maintain access once SCIM is enabled.
+{{< /admonition >}}
+
+For users who already exist in the Grafana instance:
+
+- SCIM matches them by email address
```

Comment:
SCIM doesn't match them by email address. I'd rephrase this with something like

```
- IdP looks up the user by filtering all the users using the user's login and the Unique identifier field for users - configurable at IdP
- The IdP expect a single result from Grafana
- The IdP learns the relationship between the user found at Grafana and the ID that Grafana uses
- The IdP updates Grafana with the External ID
- With the newly provisioned user, Grafana updates the authentication validations
- Grafana will expect the SAML integration to return the same External ID from the SAML assertions to validate the logged in user is the same as the provisioned user
```

This is the overall idea of how the validation works

Reply:
Thanks for the feedback, can you please check this commit https://github.com/grafana/grafana/pull/102478/commits/82fbe40a30ed8c1572886e673758acb538c0b22b ?

I've modified the section a bit to match to the provided feedback.